packages/arb-token-bridge-ui/tests/e2e/specs/approveToken.cy.ts

```diff
@@ -18,37 +17,50 @@ describe('Approve token and deposit afterwards', () => {
 
       // Select the ERC-20 token
       cy.findByText('Added by User').should('exist')
-      cy.findByText(ERC20TokenName).click()
+      cy.findByText(ERC20TokenName).trigger('click')
```

Comment:
Why are some places using `.click()` whereas this is using `trigger(click)`?

Reply:
Agree we should use `.click` and if we need to trigger the event we should leave a comment.

Although we really shouldn't use trigger